aries-site/src/examples/components/button/DosDonts/ButtonGoodCancelPreview.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Box, Button, Heading, Header, Text, Spinner } from 'grommet';
+
+export const ButtonGoodCancelPreview = () => (
+  <Box width="large" gap="medium" pad="small">
+    <Header
+      direction="column"
+      align="start"
+      gap="xxsmall"
+      pad={{ horizontal: 'xxsmall' }}
+    >
+      <Heading level={2} margin="none">
+        Rule Ordering
+      </Heading>
+      <Text>
+        This rule set is for manipulating ASM/ARM field in departments and
+        products. Also changing units to add chargeback information
+      </Text>
+    </Header>
+    <Box gap="small" align="center" justify="center">
+      <Spinner size="medium" />
```

Comment:
Why are we rendering a spinner here as opposed to just showing some demo set of rules? I find the spinner distracting/confusing.

Reply:
Keeth is going to update designs so I will fix with her updates